### ISSUE TITLE
 Pediatric stroke

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6643]
**Preferred term label:**
 Pediatric stroke

**Synonyms**
Childhood stroke

**Definition (free text, please give PubMed ID)**
Stroke in children;  PMID: 22254140

**Parent term (use hpo.jax.org/app)**
Stroke

**Comment by pnrobinson** [comment]
To annotate a child with stroke, we would use the HPO term for stroke together with this term for onset (https://hpo.jax.org/app/browse/term/HP:0410280) or if we were using a phenopacket to record the data, we could use one of the options for recording age (https://phenopackets-schema.readthedocs.io/en/latest/).

In general, we try to avoid making combinations like this, because that would mean we would need to add similary modifiers to thousands of other terms which would make the HPO very difficult to use.
